carbonmark-data/components/Layout/style.module.scss

```diff
@@ -116,7 +116,8 @@
 
 .mobileBottomNavItem {
   display: flex;
-  height: 44px;
+  padding: 0.4rem;
+  height: 3.6rem;
```

Comment:
Why is this not 4.4rem?

Reply:
Unless I'm missing something:
`3.6 + 0.4 + 0.4 = 4.4`

Reply:
This is from figma

<img width="212" alt="grafik" src="https://github.com/KlimaDAO/klimadao/assets/138768831/1ddbab04-57da-4e67-8724-2d731decfe03">